lambdas/common-layer/nodejs/node_modules/dev-portal-common/notify-cfn.js

```diff
@@ -100,7 +100,7 @@ function ofSuccess (config) { return notifyCFN({ ...config, responseStatus: 'SUC
  */
 function ofFailure (config) { return notifyCFN({ ...config, responseStatus: 'FAILED' }) }
 
-notifyCFN.ofSuccess = ofSuccess
-notifyCFN.ofFailure = ofFailure
-
-module.exports = notifyCFN
+module.exports = {
+  ofSuccess,
+  ofFailure
+}
```

Comment:
Was there anything that used the raw `notifyCFN`?

Reply:
Nope, and that's why I changed it. I'd rather have fewer ways to call it so it's fewer things we need to audit later.